### PR TITLE
Trading view fix

### PR DESCRIPTION
**Pull request by ssakone**
# Various fix for trading view.
 
 ### need to check : Wrong order of animation steps on widget enlargement [1021](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1021)
- [x]  Work
- [ ]  Not Work (Re-Check)

 ### need to check : Settings icon don't do anything on simple view [1020](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1020)
- [ ] Work
- [ ] Not Work (Re-Check)

 ### need to check : Simple trading view widget flickering on view opening from another tab [1019](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1019)
- [x] Work
- [ ] Not Work - (Re-Check)

 ### need to check : Search input clearing on tab switch but base/rel ticker tables still filtering until you search again [1018](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1018)
- [x] Work
- [ ] Not Work (Re-Check)

 ### need to check : Recover funds button do not produce output on simple trading view [1016](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1016)

- [x] Work
- [ ] Not Work (Re-Check)

 ### need to check : Amounts doesn't fit selectors in pro view in some cases [1004](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1004)

- [x] Work
- [ ] Not Work (Re-Check)

### CODE REVIEW

**Review by SylEze — CHANGES_REQUESTED** [review]
Please fix spaces between operators.

**Comment by ssakone** [comment]
> Wrong coding style, please fix everything.

**Comment by ssakone** [comment]
> Please fix spaces between operators.

okay

File: atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml [hunk 1]
```diff
@@ -16,6 +16,7 @@ Item {
     readonly property var subPages: Main.getSubPages()
 
     // Variable which holds the current sub-page of the SimpleView.
+    property string recover_funds_result: '{}'
```
Comment (SylEze) [inline atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml:19]: camelCase

File: atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml [hunk 2]
```diff
@@ -166,10 +167,14 @@ Item {
...
-                    Trade
-                    {
-                        id: simple_trade
-                        width: parent.width
+                    ClipRRect {
+                        anchors.fill: parent
+                        radius: 20 
+                        Trade
+                        {
+                            id: simple_trade
```
Comment (SylEze) [inline atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml:175]: _camelCase

File: atomic_defi_design/qml/Exchange/Trade/SimpleView/Trade.qml [hunk 3]
```diff
@@ -791,17 +803,20 @@ ClipRRect // Trade Card
...
+        Item // Swap Info - Details
         {
             id: _feesCard
             anchors.horizontalCenter: parent.horizontalCenter
             width: 350
             height: 60
 
             enabled: !_swapAlert.visible
-            visible: enabled & !bestOrderSimplified.visible & !coinSelectorSimplified.visible
+            visible: _feesList.count!==0 & _tradeCard.selectedOrder!==undefined &  parseFloat(_fromValue.field.text) > 0 & !bestOrderSimplified.visible & !coinSelectorSimplified.visible
```
Comment (SylEze) [inline atomic_defi_design/qml/Exchange/Trade/SimpleView/Trade.qml:814]: Spaces

**Review by SylEze — CHANGES_REQUESTED** [review]

File: atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml (hunk 1 above)
Comment (SylEze) [reply atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml:19]: Also the comment is misplaced, it should be on the upper one property, don't know why it's there now

Comment (ssakone) [reply atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml:19]: old code needed by refund, not writted by me, it will be change later.

**Comment by smk762** [comment]
> # Various fix for trading view.
> ### need to check : Wrong order of animation steps on widget enlargement [1021](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1021)
> * [x]   Work
> * [ ]   Not Work (Re-Check)
> 
> ### need to check : Settings icon don't do anything on simple view [1020](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1020)
> * [ ]  Work
> * [ ]  Not Work (Re-Check)
> 
> ### need to check : Simple trading view widget flickering on view opening from another tab [1019](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1019)
> * [x]  Work
> * [ ]  Not Work - (Re-Check)
> 
> ### need to check : Search input clearing on tab switch but base/rel ticker tables still filtering until you search again [1018](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1018)
> * [x]  Work
> * [ ]  Not Work (Re-Check)
> 
> ### need to check : Recover funds button do not produce output on simple trading view [1016](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1016)
> * [ ]  Work
> * [ ]  Not Work (Re-Check)
> 
> ### need to check : Amounts doesn't fit selectors in pro view in some cases [1004](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1004)
> * [x]  Work
> * [ ]  Not Work (Re-Check)

`Settings icon don't do anything on simple view` - works in sidebar menu in simple mode, but was this intended to be in header also? - https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1020#issue-928054478

`Recover funds button do not produce output on simple trading view` - Will let @tonymorony check to confirm this unless I can find a wallet with recovery funds or generate a fail leading to it.

All other cases sighted and confirmed working.

**Comment by ssakone** [comment]
> > # Various fix for trading view.
> > ### need to check : Wrong order of animation steps on widget enlargement [1021](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1021)
> > 
> > * [x]    Work
> > * [ ]    Not Work (Re-Check)
> > 
> > ### need to check : Settings icon don't do anything on simple view [1020](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1020)
> > 
> > * [ ]   Work
> > * [ ]   Not Work (Re-Check)
> > 
> > ### need to check : Simple trading view widget flickering on view opening from another tab [1019](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1019)
> > 
> > * [x]   Work
> > * [ ]   Not Work - (Re-Check)
> > 
> > ### need to check : Search input clearing on tab switch but base/rel ticker tables still filtering until you search again [1018](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1018)
> > 
> > * [x]   Work
> > * [ ]   Not Work (Re-Check)
> > 
> > ### need to check : Recover funds button do not produce output on simple trading view [1016](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1016)
> > 
> > * [ ]   Work
> > * [ ]   Not Work (Re-Check)
> > 
> > ### need to check : Amounts doesn't fit selectors in pro view in some cases [1004](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1004)
> > 
> > * [x]   Work
> > * [ ]   Not Work (Re-Check)
> 
> `Settings icon don't do anything on simple view` - works in sidebar menu in simple mode, but was this intended to be in header also? - [#1020 (comment)](https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1020#issue-928054478)
> 
> `Recover funds button do not produce output on simple trading view` - Will let @tonymorony check to confirm this unless I can find a wallet with recovery funds or generate a fail leading to it.
> 
> All other cases sighted and confirmed working.

not just hide it on simple_view and how to trade button, faq,
i m hiding now the two left butto.

**Comment by tonymorony** [comment]
`Recover funds button do not produce output on simple trading view - Will let @tonymorony check to confirm this unless I can find a wallet with recovery funds or generate a fail leading to it.` can confirm that refund popup works on my side now

**Review by tonymorony — COMMENTED** [review]
lgtm, thanks

**Review by tonymorony — DISMISSED** [review]

**Review by smk762 — DISMISSED** [review]

**Comment by tonymorony** [comment]
`Conflicting files
atomic_defi_design/qml/Exchange/Trade/SimpleView/Trade.qml` there is a conflict with dev now @ssakone , please check

**Pull request merged**